queries.md

```diff
@@ -16,47 +16,165 @@ SELECT \* FROM users;
 Using the `movies_db` database, write the correct SQL queries for each of these tasks:
 
 1.  The title of every movie.
+```
+SELECT title 
+    FROM movies;
+```
 
 2.  All information on the G-rated movies.
+```
+SELECT * 
+    FROM movies 
+    WHERE rating = 'G';
+```
 
 3.  The title and release year of every movie, ordered with the
     oldest movie first.
+```
+SELECT title, release_year 
+    FROM movies 
+    ORDER BY release_year;
+```
     
 4.  All information on the 5 longest movies.
+```
+SELECT *
+    FROM movies 
+    ORDER BY runtime DESC
+    LIMIT 5;
+```
 
-5.  A query that returns the columns of `rating` and `total`, tabulating the
-    total number of G, PG, PG-13, and R-rated movies.
+5.  A query that returns the columns of `rating` and `total`, tabulating the total number of G, PG, PG-13, and R-rated movies.
+```
+SELECT rating, COUNT(*) AS total
+    FROM movies 
+    GROUP BY rating
+    ORDER BY rating;
```

Comment:
Not necessary and might be a performance hit if there are many ratings (probably only if # ratings > # movies).
```suggestion
```